crates/rolldown/tests/rolldown/misc/object_shorthand_property/artifacts.snap

```diff
@@ -37,5 +37,5 @@ assert.strictEqual(value.foo, "foo");
 (5:19) "value." --> (8:19) "value."
 (5:25) "foo, " --> (8:25) "foo, "
 (5:30) "'foo')" --> (8:30) "\"foo\")"
-(5:36) "" --> (8:36) ";\n"
+(5:36) [invalid] --> (8:36)
```

Comment:
NICE, we finally have a way to see what's wrong without source maps.

Reply:
Interesting, I thought now "[invalid]" means previously it was panic, but it looks that wasn't always the case.
I think this column 36 didn't panic because the slice `v[36..36]` doesn't cause out of range for the vec of length 36.

Reply:
Let's merge it first.